src/templates/one/content.js

```diff
@@ -0,0 +1,37 @@
+const I18n = require('frontend-i18n');
+
+const getTranslationsForSite = siteId => {
+  let filePath = '../../../translations/';
+  switch (siteId) {
+    case 'MLA':
+      filePath += 'es-AR/';
+      break;
+    case 'MLB':
+      filePath += 'pt-BR/';
+      break;
+    default:
+      break;
```

Comment:
Maybe return an empty object with translations, but this way should break the following `require`.

Reply:
`translations/messages.json` is the default file with all keys and no translations. So it will return the neutral key

Reply:
You're right! My bad, I forget about this file